spec.bs

```diff
@@ -680,9 +686,15 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
       1. [=list/For each=] |ig| of |groups|:
         1. TODO: create bidder script runner. If failed, [=iteration/continue=].
         1. TODO: Let |interestGroup| be ... from |ig| ... minus priority and prioritySignalsOverrides and any browser-defined pieces
-        1. TODO: Set |browserSignals|'s "joinCount" to |ig|'s ...need to add and maintain joinCount in interest groups
-        1. TODO: Set |browserSignals|'s "bidCount" to |ig|'s ...need to add and maintain bidCount in interest groups
-        1. TODO: Set |browserSignals|'s "prevWins" to |ig|'s ...need to add and maintain prevWins in interest groups
+        1. Set |browserSignals|'s "joinCount" to the sum of |ig|'s

```

Comment:
Will we eventually have IDL defined for whatever class browserSignals is and joinCount, etc? We should make sure to link those once we do.

Reply:
We could; I normally associate WebIDL with defining callee APIs rather than caller APIs.  Looks like other worklets use IDL for defining some of their inputs, e.g. [PaintSize](https://drafts.css-houdini.org/css-paint-api/#paintsize) is one of the inputs [passed](https://drafts.css-houdini.org/css-paint-api/#invoke-a-paint-callback) to PaintWorklets.  Unfortunately we have four different `browserSignals` formats for each kind of FLEDGE script.  Sounds like something we should think about adding after we finish up the fundamentals.

Reply:
Yeah - totally not a blocker.